src/App.js

```diff
@@ -7,51 +7,64 @@ class App extends Component {
   render() {
     return (
       <section className="container-fluid image-lukasz">
-        <div className="row">
+        <div className="row ">
```

Comment:
Czy to jest jakas konwencja, że Często zostawiasz pustą spację za klasą?

Reply:
Done